package/MDAnalysis/core/groups.py

```diff
@@ -3270,20 +3270,31 @@ def split(self, level):
         return [self[levelindices == index] for index in
                 unique_int_1d(levelindices)]
 
-    def guess_bonds(self, vdwradii=None):
-        """Guess bonds that exist within this :class:`AtomGroup` and add them to
-        the underlying :attr:`~AtomGroup.universe`.
+    def guess_bonds(self, vdwradii=None, fudge_factor=0.55, lower_bound=0.1,
+                    **kwargs):
+        """Guess bonds, angles, and dihedrals between the atoms in this
+        :class:`AtomGroup` and add them to the underlying
+        :attr:`~AtomGroup.universe`.
 
         Parameters
         ----------
         vdwradii : dict, optional
             Dict relating atom types: vdw radii
 
+        fudge_factor : float, optional
```

Comment:
Could you add some tests that check that these new parameters are being passed properly through?

Reply:
should I make a `TestGuessBonds` class and test `guess_bonds` for different values for fudge_factor, lower_bond etc?

Reply:
Oh, found TestGuessBonds class I will add these tests to it 

    def test_fudge_factor(self):
        u = mda.Universe(two_water_gro, guess_bonds=True, fudge_factor=0.5)
        self._check_universe(u)
        assert u.kwargs['guess_bonds']
        assert_equal(0.5, u.kwargs['fudge_factor'])

    def test_lower_bound(self):
        u = mda.Universe(two_water_gro, guess_bonds=True, lower_bound=0.8)
        self._check_universe(u)
        assert u.kwargs['guess_bonds']
        assert_equal(0.8, u.kwargs['lower_bound'])

So, I should add these attributes to Universe class too...